resources/js/components/structures/PageEditor.vue

```diff
@@ -186,9 +192,9 @@ export default {
 
             this.$axios.post(postUrl, {
                 type: this.type,
-                values: this.values
+                values: this.visibleValues
             }).then(response => {
-                this.$emit('submitted', this.values);
+                this.$emit('submitted', this.visibleValues);
```

Comment:
I think this needs to remain as `this.values` so _all_ the values get included in the event.